source/shared/net/desktop-wallet/send-shielded-amount.rst

```diff
@@ -1,8 +1,8 @@
 .. _send-shielded-amount:
 
-========================
-Make a shielded transfer
-========================
+====================================
+Send and receive a shielded transfer
```

Comment:
Depending on whether or not the subtitle below is changed or not, this might have to be updated as well.